shell/app/common/components/table/index.tsx

```diff
@@ -60,7 +60,7 @@ function WrappedTable<T extends object = any>({
   pagination: paginationProps,
   onChange,
   slot,
-  dataSource = [],
+  dataSource,
```

Comment:
I dont understand why remove this default value. Then leads code below has to be like `dataSource?.length || 0`